docs/reference/indices/split-index.asciidoc

```diff
@@ -0,0 +1,165 @@
+[[indices-split-index]]
+== Split Index
+
+number_of_routing_shards
```

Comment:
I thought this settings was temporary?

Reply:
yes but I need to document it, it will be here for the entire 6.x series

Reply:
Can you elaborate on why it has to be there in 6.x and go in 7.0?

Reply:
the reason is that if we make it automatic it changes the distribution of docs which is a bwc break and we shouldn't do this in a minor